src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -183,14 +183,39 @@ void grpc_sockaddr_make_wildcard6(int port,
 }
 
 std::string grpc_sockaddr_to_string(const grpc_resolved_address* resolved_addr,
-                                    bool normalize) {
```

Comment:
Any reason not to make this `absl::StatusOr<std::string>`?

Seems like it would be easier to use safely.

Reply:
Done.

I kept the current behavior (i.e. get a string in any case) by default and only changed to return an error where it looks doable, PTAL.

Also considered adding a helper that wraps (ok? value : message) but decided not to since this will be around forever once added.

Reply:
Also we have use cases in prod, so this will need to be backported manually with more changes.